plugins/devtools/src/components/Content/ScheduledTasksContent/ScheduledTasksContent.tsx

```diff
@@ -0,0 +1,257 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useState } from 'react';
+import Box from '@material-ui/core/Box';
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from '@material-ui/core/InputLabel';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import Tooltip from '@material-ui/core/Tooltip';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import { Progress, Table, TableColumn } from '@backstage/core-components';
+import Alert from '@material-ui/lab/Alert';
+import { useScheduledTasks, useTriggerScheduledTask } from '../../../hooks';
+import { TaskApiTasksResponse } from '@backstage/plugin-devtools-common';
+import { alertApiRef, configApiRef, useApi } from '@backstage/core-plugin-api';
+import RefreshIcon from '@material-ui/icons/Refresh';
+import NightsStay from '@material-ui/icons/NightsStay';
+import Error from '@material-ui/icons/Error';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import { ScheduledTaskDetailPanel } from './ScheduledTaskDetailedPanel';
+import { RequirePermission } from '@backstage/plugin-permission-react';
+import { devToolsTaskSchedulerCreatePermission } from '@backstage/plugin-devtools-common';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    paperStyle: {
+      display: 'flex',
+      marginBottom: theme.spacing(2),
+    },
+    flexContainer: {
+      display: 'flex',
+      flexDirection: 'row',
+      padding: 0,
+    },
+    formControl: {
+      minWidth: 240,
+      marginBottom: theme.spacing(2),
+    },
+    detailPanel: {
+      padding: theme.spacing(2),
+      backgroundColor: theme.palette.background.default,
+    },
+    detailLabel: {
+      fontWeight: 'bold',
+      marginRight: theme.spacing(1),
+    },
+    errorIcon: {
+      color: theme.palette.error.main,
+      marginRight: theme.spacing(1),
+      fontSize: '1.2rem',
+    },
+    detailPanelAlert: {
+      marginBottom: theme.spacing(2),
+    },
+  }),
+);
+
+const StatusDisplay = ({
+  icon,
+  text,
+}: {
+  icon: React.ReactNode;
+  text: string;
+}) => (
+  <Box display="flex" alignItems="center">
+    {icon}
+    <Typography variant="body2" style={{ marginLeft: 8 }}>
+      {text}
+    </Typography>
+  </Box>
+);
+
+/** @public */
+export const ScheduledTasksContent = () => {
+  const classes = useStyles();
+  const configApi = useApi(configApiRef);
+  const alertApi = useApi(alertApiRef);
+  const plugins = configApi.getStringArray('devTools.scheduledTasks.plugins');
```

Comment:
I'm torn on this one, having to manually add these seems a bit tedious but I'm also not aware of any API that will give you a list of all the plugins that use the scheduler. I wonder if the middle ground for now it to allow people to type in a value if it's not in the list? 🤔